packages/ui/src/components/TagList/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -94,16 +99,15 @@ exports[`TagList renders correctly 1`] = `
           class="cache-1anauza-StyledContainer el6733p2"
         >
           <span
-            aria-disabled="false"
```

Comment:
i think Text component doesn't forward aria-disabled props :)

Reply:
https://github.com/scaleway/ultraviolet/pull/3183/files#diff-f307eebe3b932f9e681fd1ceb69316f0bfb8b7929514a977fd7adcd1aba1beb4R108

Reply:
@philibea that's right but it was a mistake to put it on text only. Instead I will add it to the container so it will explicitly say that the container and it's children are disabled.

Reply:
<img width="2560" alt="Screenshot 2023-11-23 at 10 54 14" src="https://github.com/scaleway/ultraviolet/assets/15812968/1fa0eff7-fda8-4425-9bc8-7c85645445f6">